src/attributes.jl

```diff
@@ -332,6 +332,18 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    RequiredBridges()
+
+A list of bridges that are required when using the optimizer. For instance, if
```

Comment:
What does "required" mean? Required to pass tests? What does this imply for direct mode?

Reply:
> What does "required" mean? Required to pass tests?

I was hesitant for the naming, may be recommended (and RecommendedBridges) is better ?

> What does this imply for direct mode?

If the user uses a `LazyBridgeOptimizer` in direct mode then the bridges will be automatically added. The bridges are added in the `LazyBridgeOptimizer` constructor, not just in the `full_bridge_optimizer` function.

Reply:
I don't think `RecommendedBridges` is any clearer. Recommended for what? How is someone looking at this description supposed to know how it fits into `LazyBridgeOptimizer`?

If a solver has required bridges, maybe it shouldn't publicly return its MOI optimizer and instead return a `LazyBridgeOptimizer` that wraps the optimizer with the required bridges? It doesn't make sense to me to have required bridges while the default way of constructing the solver doesn't include them.

Reply:
There is indeed the other option of returning the optimizer wrapped in a `SingleBridgeOptimizer` (like we currently do in tests). As the bridge is required there is no need to include it in the Bellman-Ford computation.
What would be the recommendation for naming the constructor and the optimizer ?
Adding an underscore before `Optimizer` ?
```julia
struct _Optimizer <: MOI.AbstractOptimizer
end
Optimizer(...) = CustomBridge(_Optimizer(...))
```

Reply:
> Adding an underscore before `Optimizer` ?

Yes, that makes sense.